### DISCUSSION TITLE
功能：禁止用户拖拽内容进编辑器

### DISCUSSION THREAD

**Discussion by xixihhhh** [wangeditor-next/wangEditor-next#303] [Q&A] [answered]
### 需求描述
因为我是用了customPaste过滤用户的输入，但是用户可以拖拽一些文字图片进编辑器，拖拽的内容不会通过customPaste过滤，像请教一下怎么禁止拖拽内容进入编辑器
### 相关 issue
https://github.com/wangeditor-team/wangEditor/issues/5972

**Comment by cycleccc** [comment]
https://github.com/cycleccc/wangEditor-next/blob/1f42f9c5b6710c67fb449d9bb75d044bdc84f6d8/packages/core/src/text-area/event-handlers/drop.ts#L13
拖拽的逻辑在这里处理，针对你的这个情况可能需要另开放一个配置控制禁止拖拽。

**Comment by cycleccc** [comment, marked as answer]
根据容器 id 自行禁止拖拽事件如：
~~~ JavaScript
document.getElementById('editor-text-area').addEventListener(
  'drop',
  (e) => {
    e.preventDefault();
    e.stopPropagation();
  },
  { capture: true }
);
document.getElementById('editor-text-area').addEventListener(
  'dragover',
  (e) => {
    e.preventDefault();
    e.stopPropagation();
  },
  { capture: true }
);
~~~

**Reply by liuchuzhang** [reply]
it works, thank you. i want v you 50 KFC @cycleccc
